extensions/ql-vscode/src/remote-queries/remote-queries-interface.ts

```diff
@@ -111,6 +111,7 @@ export class RemoteQueriesInterfaceManager {
           localResourceRoots: [
             Uri.file(this.analysesResultsManager.storagePath),
             Uri.file(path.join(this.ctx.extensionPath, 'out')),
+            Uri.file(path.join(this.ctx.extensionPath, 'node_modules/@vscode/codicons/dist')),
```

Comment:
I feel like this part should be dealt by webpack somehow. @aeisenberg what do you think?

The reason I mention webpack is because webpack is the tool that is building up our "executable" and sticks it in the `out` directory. It'd be cool if it included any codicons resources in it too.

Reply:
It might be that without extra webpack instructions, this file won't exist when the extension is built.

Reply:
I couldn't find any examples of other icons/things that we require from node_modules 😿 The other ones ([like the primer/octicons](https://github.com/github/vscode-codeql/blob/9c29c5c9c6b7c9f073034f79118f647138552e0b/extensions/ql-vscode/src/remote-queries/view/RepoListCopyButton.tsx#L4)) are generally just components that you can import 🤔 


Also, good call about building/installing the extension properly! I downloaded an actions build from https://github.com/github/vscode-codeql/actions/runs/2657604754 and the codicons aren't visible 🤦🏽 

I'll take a quick look at webpack - though I may need a hand to make changes 😅

Reply:
Now that I've [moved `codicons` to the regular `dependencies`](https://github.com/github/vscode-codeql/pull/1424/commits/9669cd850a5e84516e318f0107a7dc0464010e0c) (instead of `devDependencies`), the current approach seems to work fine 😌 

Let me know if we still need/want to change webpack! Otherwise, this should be good to go 🤞🏽

Reply:
Lets get it merged and we can play around with webpack some other time.